src/components/TodoError/TodoError.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  error: string,
+  setError: (value: string) => void,
```

Comment:
Consider using a more descriptive name for the 'setError' function, such as 'handleError' or 'onError', to better indicate its purpose.

Reply:
done

Reply:
done